applications/serial_lte_modem/doc/CMUX_AT_commands.rst

```diff
@@ -58,8 +58,8 @@ Syntax
    AT#XCMUX[=<AT_channel>]
 
 The ``<AT_channel>`` parameter is an integer used to indicate the address of the AT channel.
-If specified, it must be between 1 and the total number of channels.
 The AT channel denotes the CMUX channel where AT data (commands, responses, notifications) is exchanged.
+If specified, it must be 1, and if :ref:`PPP <CONFIG_SLM_PPP>` is enabled it can also be 2 (to allocate the first CMUX channel to PPP).
```

Comment:
```suggestion
If specified, it must be 1, unlessf :ref:`PPP <CONFIG_SLM_PPP>` is enabled.
If PPP is enabled, it can also be 2 (to allocate the first CMUX channel to PPP).
```